packages/loader/container-loader/src/container.ts

```diff
@@ -1428,7 +1449,10 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
                 this._audience.removeMember(leftClientId);
             }
         } else {
-            const local = this._clientId === message.clientId;
+            const local = this.clientIdHistory.includes(message.clientId);
+            if (local && this._clientId !== message.clientId) {
+                this.logger.sendErrorEvent({ eventName: "matchedOldClientIdInSignal" });
```

Comment:
Same as above, is there any additional info that we could include on this event to make it more actionable?

Reply:
signals are not ordered, so this check doesn't apply here. i don't think we should do anything for signals

---
In reply to: [488089587](https://github.com/microsoft/FluidFramework/pull/3613#discussion_r488089587) [](ancestors = 488089587)